omnigan/data.py

```diff
@@ -149,25 +143,30 @@ def __getitem__(self, i):
         """
         paths = self.samples_paths[i]
 
-        
         if self.transform:
-            return {
+            item = {
                 "data": self.transform(
                     {task: pil_image_loader(path, task) for task, path in paths.items()}
                 ),
                 "paths": paths,
                 "domain": self.domain,
                 "mode": self.mode,
             }
+            if "d" in item["data"].keys():
+                item["data"]["d"] = get_normalized_depth(item["data"]["d"], self.domain)
```

Comment:
can't this be in `self.transform` and `pil_image_loader`? since we have the path and the task?

Reply:
I had this in pil_image_loader first but then I don't want it to return and Image. if you think that makes more sense I can put it back there and return a tensor for depth, else an image

Reply:
Oh now I remember why I can't put it in the pil image loader. It's because the transforms were written for Image objects and it's just more convenient to apply the same on all things of your item